mobile/src/src/components/Menu/Profile/ProfileComponents.jsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import { StyleSheet, Text, View } from "react-native";
+
+const ProfileComponents = ({ label, val, row }) => {
+  const isRow = ({ label, val, row }) => {
+    if (row) {
+      if (val == "" || val === null) {
+        return (
+          <View>
+            <Text style={styles.textLabelRow}>{label}</Text>
+            <Text style={styles.textValRowValNull}>
+              Edit to add your {label}
+            </Text>
+          </View>
+        );
+      } else {
+        return (
+          <View>
+            <Text style={styles.textLabelRow}>{label}</Text>
+            <Text style={styles.textValRow}>{val}</Text>
+          </View>
+        );
+      }
+    } else {
+      if (val == "" || val === null) {
```

Comment:
Why is this if case repeated?

Reply:
the first one is if the field is on a row and the second one is if it's not on a row

Reply:
ah gotcha, thats my bad